examples/step-90/doc/builds-on

```diff
@@ -0,0 +1 @@
+step-85
```

Comment:
You also need to add the tutorial to https://github.com/dealii/dealii/blob/master/doc/doxygen/tutorial/tutorial.h.in.

Could you also add some references, e.g., to you paper?